void printException(dynamic error, StackTrace stackTrace) =>
    print("Exception occured: $error stackTrace: $stackTrace");
